Makefile

```diff
@@ -1,56 +1,67 @@
+# Default to using regular `cargo`. The `ci` target will override this.
+VICEROY_CARGO=cargo
+export VICEROY_CARGO
```

Comment:
As I guess we are just replacing this var in the Makefile context (but not being used by child processes that makes the env var a requirement), maybe we don't need to export it?

Reply:
Correct! 87b48ef